### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by kitpao**
In this branch we made a functional code implementing the game logic

### CODE REVIEW

File: lib/game_methods.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    self.move = 0
+    self.selection = []
+  end
+
+  def winner(selection)
+    possibilities = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]]
+    sorted = selection.sort
+    possibilities.each do |x|
+      if sorted & x == x
+        puts "CONGRATS!!!! #{name.upcase}.. You are the winner!!!!!!"
```
Comment (wathigo) [inline lib/game_methods.rb]: **[REQUIRED]**
* All user interaction code like `puts` and `print` should only reside in `main.rb`. See [this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png)

File: bin/main.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
-require 'timeout'
+
+require '../lib/game_methods'
```
Comment (wathigo) [inline bin/main.rb]: **[OPTIONAL]** 
* Let's use `require_relative` as opposed to `require` since it's the ruby convention of importing files.

**Review by wathigo — CHANGES_REQUESTED** [review]
# Changes Required 
## Hi Devs :wave: 
I have enjoyed playing the game.
Let's fix a few issues before we move on.
See the comments :point_down: 

**After implementing the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

Hi team!

There are few updates to make, although you have implemented the changes requested from my colleague.

- Use attr_readers/attr_writers instead of attr_accessors.  By using attr_accessors you allow writing that property from outside the class. That variable can be set to anything.  [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)

- Create seperate files for your player class and game class, in lib folder.

After implementing the changes, submit another form for a code review.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Review by uche-inyama — APPROVED** [review]
## Approved

Hi team, everything looks almost fine, however, I will go ahead and approve this mile-stone. 

There is an attr_accessor in your project. I want to believe you have taken note of the point I made earlier about using attr_accessors.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Pull request merged**